src/js/components/VoterEmailAddressEntry.js

```diff
@@ -0,0 +1,13 @@
+import React, {Component} from 'react'
+import {View, Text} from 'react-native'
+
+export default class VoterEmailAddressEntry extends Component {
+
+  render(){
+    return(
+      <View>
+      <Text>Hello</Text>
```

Comment:
VoterEmailAddressEntry needs some work with the forms, I just made a dummy page since it's required by the SignIn page for now.